inventory/hosts

```diff
@@ -153,6 +162,7 @@ de-prod
 [america:children]
 ca-prod
 us-prod
+br-prod
```

Comment:
these lists are for servers managed by the global team. we use these groups to deploy regularly. br is not managed by the global team so it should not be in this list.